### ISSUE TITLE
How to concatenate knowledge graphs embeddings with Textual embeddings by Keras embedding layers and feed to Multi perceptron layer?

### ISSUE THREAD

**Issue by maya-25** [keras-team/keras#16115]
I have generated knowledge graph embeddings by RDF2Vec(<s,p,o> RDF triples.ttl file with DBpedia URIs) of size (7536, 384) and I have embeddings for my text dataset generated by Sbert of size(14169,384). Now, how do I concatenate these embeddings using Keras embedding layers? and then model.fit the concatenated embeddings with target label for training the model? Could anyone help?
**As of now, I am using this below code and  I want to add a Keras embedding layer to concatenate the above embeddings and train my model.**  
```
Model = tf.keras.models.Sequential()
Model.add(tf.keras.layers.Dense(128, activation = 'relu'))
Model.add(tf.keras.layers.Dense(2, activation = 'sigmoid'))
Model.compile(loss='sparse_categorical_crossentropy', optimizer='adam', metrics=['acc'])
history = Model.fit(concat_embeddings, train_labels.values, epochs=30, validation_split=0.1,verbose=1, batch_size=128)
```

**Comment by haifeng-jin** [comment]
@maya-25 Thanks for the question. I would suggest you to ask this question here: https://groups.google.com/g/keras-users
The GitHub issues are mainly for bug reports and feature requests. Thanks
